modin/pandas/test/utils.py

```diff
@@ -648,6 +648,7 @@ def eval_general(
     check_exception_type=True,
     raising_exceptions=None,
     check_kwargs_callable=True,
+    md_extra_kwargs=None,
```

Comment:
Can't it be passed through `kwargs`?

Reply:
It can, but it this case `md_extra_kwargs` parameter will be mixed with read_csv `kwargs`, that potentially can cause unexpected errors.

Reply:
`pandas.read_csv` has a `engine` parameter, hasn't it? Why don't you pass it to the pandas version of `read_csv`?

Reply:
That's right, `pandas.read_csv` has the same parameter, but `md_extra_kwargs` has to be used to set modin specific parameter values (for example, we are setting `engine` to `arrow` to trigger modin implementation, but if we set the same value for `pandas.read_csv` it will cause `ValueError`).